### DISCUSSION TITLE
Welcome to OpenDeck Discussions!

### DISCUSSION THREAD

**Discussion by paradajz** [shanteacontrols/OpenDeck#31] [General]
## Welcome!

Many people have been asking about some kind of OpenDeck forum where community could gather, ask questions or post their OpenDeck based builds. GitHub Discussions seem very suitable for that!

**Comment by wst3** [comment]
I just stumbled across OpenDeck - very cool idea, looks like a great implementation - I will be checking this out. 

To get started I think I will build a "front panel" for various synthesizers, both hardware and software. That will take some thought!

Thanks for all your hard work.

**Reply by paradajz** [reply]
Thank you for the kind words! Looking forward to seeing your project.

**Comment by risentide2024** [comment]
Hello. I just discovered OpenDeck today. I bought a midi controller from Etsy that requires configuration via OpenDeck. I found the Open Deck Repository on GitHub and I have no idea how to use it. I'm used to installing software with a DMX file and there is none. What do I need to run OpenDeck on my MAC? Sorry for being such a newb.

**Reply by risentide2024** [reply]
Nevermind. I figured it out. I found the version with the exe file and loaded it on a Windows computer. It took a bit of searching.
